src/bootstrap/dist.rs

```diff
@@ -226,7 +226,9 @@ fn make_win_dist(
         let idx = line.find(':').unwrap();
         let key = &line[..idx];
         let trim_chars: &[_] = &[' ', '='];
-        let value = line[(idx + 1)..].trim_start_matches(trim_chars).split(';').map(PathBuf::from);
+        let separator = if cfg!(windows) { ";" } else { ":" };
+        let value =
+            line[(idx + 1)..].trim_start_matches(trim_chars).split(separator).map(PathBuf::from);
```

Comment:
We should use [`std::env::split_paths`](https://doc.rust-lang.org/nightly/std/env/fn.split_paths.html) here instead, I think?

Reply:
I knew there is such function but searched at the wrong place...

Reply:
Thanks, tested on native Windows and cross compiled from Linux.